angu-fixed-header-table.js

```diff
@@ -74,6 +79,7 @@
     function fixedHeader($timeout, $window, $q, $rootScope){
         var attr_prefix = 'fixedHeader';
         var ATTRS = {
+            TABLE_HEIGHT : 'tableHeight', // table height
```

Comment:
Well I couldn't find any usage in viq but since this is an open source project, and the attribute was supported, we might just leave it the way it was (without the prefix).